app/src/main/java/com/example/plantsapp/presentation/PlantApplication.kt

```diff
@@ -1,14 +1,14 @@
+// TODO remove this and maybe replace with multibinding
```

Comment:
Detekt is complaining on imports for workers settings. Those workers settings should be moved somewhere:
https://github.com/Lonexera/students-lab/issues/105